boot/boot.js

```diff
@@ -498,10 +503,32 @@ $tw.utils.getTypeEncoding = function(ext) {
 	return typeInfo ? typeInfo.encoding : "utf8";
 };
 
+var polyfill =[
+	"// this polyfills the globalThis variable",
+	"// using the this variable on a getter ",
+	"// inserted into the prototype of globalThis",
+	"(function() {",
+	"	if (typeof globalThis === 'object') return;",
+	"	// node.green says this is available since 0.10.48",
+	"	Object.prototype.__defineGetter__('__wow__', function() {",
+	"		return this;",
+	"	});",
+	"	__wow__.globalThis = __wow__; // lolwat",
```

Comment:
Rather than `__wow__` could we use a descriptive variable name? If the "lolwat" comment doesn't serve a purpose then it should be removed too.